Tools/clinic/clinic.py

```diff
@@ -4865,10 +4865,21 @@ def state_parameter(self, line: str | None) -> None:
             return
 
         line = line.lstrip()
-
-        if line in ('*', '/', '[', ']'):
-            self.parse_special_symbol(line)
-            return
+        func = self.function
+        match line:
```

Comment:
This is super nitpicky, but the `line = line.lstrip()` assignment here is now redundant; you could just do:

```py
func = self.function
match line.lstrip():
    ...
```

Reply:
Sure, let's do it now that the window is open